### DISCUSSION TITLE
Built in Table Mode

### DISCUSSION THREAD

**Discussion by txtyash** [helix-editor/helix#10140] [General]
Table mode for org & markdown files similar to this plugin: [dhruvasagar/vim-table-mode](https://github.com/dhruvasagar/vim-table-mode)

**Comment by EpocSquadron** [comment]
This is really neat. It reminds me of automatic indentation, and could likely be implemented via tree-sitter in a similar way. Then it could be scoped to file types it makes sense for.

**Comment by pascalkuthe** [comment]
This can be done pretty well with multicurosrs/good use of generic editor functiality. This is too niche for core. If somebody really wants this This could be a plugin one day

**Comment by debajyoti1990** [comment]
I found a very interesting repo while searching for markdown in helix. It could be a very good contender for markdown table plugin someday.

https://github.com/susisu/mte-kernel?tab=readme-ov-file
